.ci_support/linux_.yaml

```diff
@@ -0,0 +1,2 @@
+go_compiler:
+- go
```

Comment:
This file would be removed once conda-forge/conda-forge-pinning-feedstock#95 is merged.

Reply:
This file is okay here. It was generated by `conda-smithy render.`